src/utils.ts

```diff
@@ -48,6 +48,7 @@ export function soundTypeToField(soundType: SoundType) {
 
 export function namesFromSoundDatabase(soundDb: SoundDatabase) {
     const result = Object.entries(soundDb)
-        .map(([key, value]) => ({id: key, display_name: value.display_name}));
+        .map(([key, value]) => ({ id: key, display_name: value.display_name }))
+        .sort((a, b) => a.display_name.localeCompare(b.display_name));
```

Comment:
I intentionally didn't sort this here, because I think sorting should be done at the UI level. This function is intended to just return the id, display_name pairs as they are.

For the actor sound select drop-down, it's already being sorted in the `buildNameOptions` function.